kroxylicious-operator/pom.xml

```diff
@@ -327,4 +327,143 @@
             </testResource>
         </testResources>
     </build>
+
+    <profiles>
+        <profile>
+            <id>kubeTest</id>
+            <properties>
+                <container.tool>podman</container.tool> <!-- override for docker -->
+                <container.tag>test</container.tag>
+                <image.archive.file>/tmp/myimage.tar</image.archive.file>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
```

Comment:
It is inevitable that some of the KTs need the operand image.
Would you give Maven (kroxylicious-app) the responsibility to build the operand image too (if activated with the right profile)?